.github/workflows/draft-release.yml

```diff
@@ -108,7 +111,55 @@ jobs:
           unzip -d artifact-windows artifact-windows.zip
           lifecycle_path=$(ls artifact-windows/lifecycle-*windows.x86-64.tgz)
           echo "ARTIFACT_WINDOWS_PATH=$PWD/$lifecycle_path" >> $GITHUB_ENV
+      - name: Create Pre Release
+        if: contains(env.LIFECYCLE_VERSION, 'rc')
+        id: create_pre_release
+        uses: actions/create-release@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ env.LIFECYCLE_VERSION }}
+          release_name: lifecycle v${{ env.LIFECYCLE_VERSION }}
+          draft: true
+          prerelease: true
+          body: |
```

Comment:
This duplication is 😐  but I don't see a way to fix it easily.

Reply:
Can we put the text in a file and stream it in twice?

Reply:
Maybe... but it gets tricky because we're also interpolating variables in the text.

Reply:
I'm ok with keeping it as is but maybe if we're doing that, it's worth adding a comment next to each `body` so if we decide in the future to edit this text, we won't forget to do it in both places.

Reply:
Concluded that improving this flow isn't worth the effort at this time.